providers/azure/README.md

```diff
@@ -272,6 +274,133 @@ To verify that virtual kubelet has started, run:
 ```cli
   kubectl --namespace=default get pods -l "app=virtual-kubelet-virtual-kubelet"
 ```
+##  Create an AKS cluster with VNet
+
+  Run the following commands to create an AKS cluster with a new Azure virtual network. Also, create two subnets. One will be delegated to the cluster and the other will be delegated to Azure Container Instances. 
+
+### Create an Azure virtual network and subnets 
+
+  First, set the following variables for your VNet range and two subnet ranges within that VNet. The following ranges are recommended for those just trying out the connector with VNet. 
+
+  **Bash**
+  ```cli
+    export VNET_RANGE=10.8.0.0/8  
+    export CLUSTER_SUBNET_RANGE=10.240.0.0/16 
+    export ACI_SUBNET_RANGE=10.241.0.0/16 
+    export VNET_NAME=myAKSVNet 
+    export CLUSTER_SUBNET_NAME=myAKSSubnet 
+    export ACI_SUBNET_NAME=myACISubnet 
+    export AKS_CLUSTER_RG=myresourcegroup 
+  ```
+  Run the following command to create a virtual network within Azure, and a subnet within that VNet. The subnet will be dedicated to the nodes in the AKS cluster.
+
+    ```cli
+    az network vnet create \
+    --resource-group $AKS_CLUSTER_RG \
+    --name $VNET_NAME \
+    --address-prefixes $VNET_RANGE \
+    --subnet-name $CLUSTER_SUBNET_NAME \
+    --subnet-prefix $CLUSTER_SUBNET_RANGE
+    ```
+
+Create a subnet that will be delegated to just resources within ACI, note that this needs to be an empty subnet, but within the same VNet that you already created. 
+
+```cli
+az network vnet subnet create \
+    --resource-group $AKS_CLUSTER_RG \
+    --vnet-name $VNET_NAME \
+    --name $ACI_SUBNET_NAME \
+    --address-prefix $ACI_SUBNET_RANGE
+```
+
+### Create a service principal
+
+Create an Azure Active Directory service principal to allow AKS to interact with other Azure resources. You can use a pre-created service principal too. 
+
+```cli
+az ad sp create-for-rbac --skip-assignment
```

Comment:
better to give a name so that we don't get a random name like "azure-cli-2018-08-29-22-29-29" below